src/main/java/com/xpanxion/java/springboot/da1/demo/service/student5/DataAccessStudent5.java

```diff
@@ -0,0 +1,99 @@
+package com.xpanxion.java.springboot.da1.demo.service.student5;
+
+import com.xpanxion.java.springboot.da1.demo.model.student5.Book;
+import com.xpanxion.java.springboot.da1.demo.model.student5.InventoryItem;
+import com.xpanxion.java.springboot.da1.demo.model.student5.Manager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class DataAccessStudent5 {
+
+    //
+    // Data Members
+    //
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    @Autowired
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+
+    private final String SELECT_BOOKSTORE_BOOKS = "SELECT * FROM bookstore_book WHERE bookstore_id = ?";
```

Comment:
My OCD likes to group all of these types of DB actions together...

 - select
 - insert
 - update